packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStylesComfyRefresh.ts

```diff
@@ -0,0 +1,232 @@
+  zIndex: overlayZIndex!,
+  overflow: 'visible',
+  opacity: 1,
+  width: 'auto',
+});
+
+export const chatMessageStylesComfyRefresh: ComponentSlotStylesPrepared<
+  ChatMessageStylesProps,
+  ChatMessageVariables
+> = {
+  root: ({ props: p, variables: v, theme }): ICSSInJSStyle => {
+    const borderFocusStyles = getBorderFocusStyles({
+      borderRadius: 'inherit',
+      variables: theme.siteVariables,
+    });
+
+    // Fixes the bubble focus border rendering on top of the user avatar
+    delete (borderFocusStyles[':focus-visible'][':before'] as ICSSInJSStyle).zIndex;
+    delete (borderFocusStyles[':focus-visible'][':after'] as ICSSInJSStyle).zIndex;
+
+    return {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: p.mine ? 'flex-end' : 'flex-start',
+      outline: 'none',
+
+      [`&:focus-visible .${chatMessageSlotClassNames.timestamp}`]: {
+        opacity: 1,
+      },
+
+      [`&:focus-visible .${chatMessageSlotClassNames.bubble}`]: borderFocusStyles[':focus-visible'],
+    };
+  },
+
+  header: ({ props: p, theme }): ICSSInJSStyle => ({
+    display: 'flex',
+    width: '100%',
+    justifyContent: p.mine ? 'flex-end' : 'start',
+    gap: pxToRem(8),
+    '& > div': {
+      paddingTop: pxToRem(8),
+    },
+    color: theme.siteVariables.colorScheme.default.foreground2,
+  }),
+
+  author: ({ props: p }): ICSSInJSStyle => ({
+    ...((p.mine || p.attached === 'bottom' || p.attached === true) && (screenReaderContainerStyles as ICSSInJSStyle)),
+    fontWeight: 400,
+    marginBottom: pxToRem(2),
+    overflow: 'hidden',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+  }),
+
+  timestamp: ({ variables: v }): ICSSInJSStyle => {
+    const styles: ICSSInJSStyle = {
+      display: 'inline-block',
+      alignSelf: 'self-start',
+      whiteSpace: 'nowrap',
+      opacity: 0,
+    };
+    if (v.hasReducedHorizontalSpace) {
+      styles.fontSize = '1rem';
+      styles.marginTop = pxToRem(3);
+      styles.marginLeft = pxToRem(2.5);
+      styles.marginRight = pxToRem(2.5);
+      styles.marginBottom = 0;
+    }
+    return styles;
+  },
+
+  body: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    display: 'flex',
+    flexDirection: p.mine ? 'row-reverse' : 'row',
+    position: 'relative',
+    maxWidth: '100%',
+    ...(p.fullWidth && {
+      width: '100%',
+    }),
+
+    ...(!p.mine &&
+      v.hasReducedHorizontalSpace && {
+        marginRight: pxToRem(16),
+      }),
+  }),
+
+  bubble: ({ props: p, variables: v, theme }): ICSSInJSStyle => {
+    return {
+      position: 'relative',
+      border: v.border,
+      borderRadius: pxToRem(6),
+      paddingLeft: pxToRem(16),
+      paddingRight: pxToRem(16),
+      paddingTop: pxToRem(8),
+      paddingBottom: p.hasReactions ? pxToRem(10) : pxToRem(8),
+
+      backgroundColor: p.mine ? `var(--chat-bubble-bg-mine, ${v.backgroundColorMine})` : v.backgroundColor,
```

Comment:
From where comes `--chat-bubble-bg-mine`? Is it defined on our side?

Reply:
Ah, this is expected to come from userland. Since we don't provide a way to set it, I'll remove it and demonstrate it in a prototype instead.